### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by southeo**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,26 @@
...
\ No newline at end of file
+# DataCite Publisher
+
+## Overview
+
+The DataCite Publisher acts as the intermediary between
+the [DiSSCo PID API](https://github.com/DiSSCo/handle-manager) and
+the [DataCite API](https://support.datacite.org/docs/api). The
+Publisher receives messages from the DiSSCo PID API through a Kafka Queue. These messages the
+records of 200-400 PIDs minted through the specimen ingestion process, and may either be a batch of
```
Comment (sharifX) [inline README.md]: should it be... "These messages **contain** records of 200-400 PIDs ..."

File: README.md [hunk 2]
```diff
@@ -1 +1,26 @@
...
+PIDs for Digital Specimens or Media Objects. These messages are mapped from the DiSSCo FDO Profile
+to the DataCite metadata schema. Once messages are mapped, they are sent, one at a time, to the
+DataCite API. This process informs DataCite of the minting of new DOIs that must be available to
```
Comment (sharifX) [inline README.md]: How does DataCite gets informed?

File: README.md [hunk 3]
```diff
@@ -1 +1,26 @@
...
+
+Thanks to the kafka queue, this process is done asynchronously from the rest of the ingestion
+process. The PID records are created in the DiSSCo PID API during ingestion, and are upgraded to
+DataCite APIs when the message reaches this Publisher.
+
+[Infrastructure Diagram](docs/publisher.png)
+
+## Error Handling
+
+In the event of an exception during this process, a critical error is logged. Manual action will be
```
Comment (sharifX) [inline README.md:23]: The log is stored in the DiSSCo infra?

**Review by sharifX — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (southeo) [reply README.md]: By sending a post request: 
> Once messages are mapped, they are sent, one at a time, to the DataCite API.

File: README.md (hunk 3 above)
Comment (southeo) [reply README.md:23]: Yes

**Review by sharifX — APPROVED** [review]

**Pull request merged**